.github/workflows/release.yml

```diff
@@ -76,7 +76,7 @@ jobs:
       if: ${{ steps.get_next_version.outputs.hasNextVersion == 'true' }}
       uses: docker/build-push-action@v5
       env:
-          DOCKER_REPOSITORY: ghcr.io/thenativeweb/get-next-version
+          DOCKER_REPOSITORY: ghcr.io/${{ github.repository }}
```

Comment:
Why aren't you using the `repository_owner` and the `event.repository.name` here as well (or, vice-versa, why aren't you using this field above?

Shouldn't both places be the same value?

Reply:
There is no special reason, I just replaced the "hard-coded" parts with Github variables.

- `github.event.repository.name` returns the repository name ( without owner )
- `github.repository_owner` returns the repository owner
- `github.repository` returns both with a slash

So you could use 

`${{ github.repository_owner }}/${{ github.event.repository.name }}`

too, yeah :)